modules/persistent-storage-byok.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// storage/container_storage_interface/persistent-storage-csi-azure.adoc
+// storage/container_storage_interface/persistent-storage-csi-ebs.adoc
+// storage/container_storage_interface/persistent-storage-csi-gcp-pd.adoc
+
+:_content-type: CONCEPT
+[id="byok_{context}"]
+= About "bring your own key" encryption
+
+// TP or GA???
+
+The "Bring your own key" (BYOK) feature encrypts root volumes and the default storage class.
+
+When starting up managed {product-title} clusters, you can pass a custom Key Management Service (KMS) key to the installer, and the installer and `cluster-storage-operator` encrypts the root volumes for the nodes in the cluster, and also applies this key to encrypt the default storage class.
```

Comment:
```suggestion
When starting up managed {product-title} clusters, you can pass a custom Key Management Service (KMS) key to the installation program to encrypt the root volumes for the nodes in the cluster. The installation program also encrypts the default storage class with this key.
```
Rewording/minimalism suggestion, as well as replacing "installer" with "installation program" per style guide. Up to you whether you want to specifically mention the `cluster-storage-operator` as doing the encryption.